stackinator/templates/compilers.gcc.spack.yaml

```
@@ -13,7 +13,7 @@ spack:
     reuse: false
   packages:
     gcc:
-      variants: [build_type=Release +strip]
+      variants: [build_type=Release +profiled +strip]
```

Even if it's the default, is it worth adding
```suggestion
      variants: [build_type=Release +bootstrap +profiled +strip]
```
just to make it explicit that we're letting gcc bootstrap itself.

---

Fully agree being explicit 😉